### ISSUE TITLE
Suppress Logging to Console

### ISSUE THREAD

**Issue by jnovack** [calmh/node-snmp-native#37]
The following line, is the bane of my existence.

https://github.com/calmh/node-snmp-native/blob/master/lib/snmp.js#L432

```
        console.warn('Response with unknown request ID from ' + rinfo.address + '. Consider increasing timeouts (' + age + ' ms old?).');
```

I'm not sure what's going wrong, but I'll increase the timeout to eternity and I'll still get this line.

Regardless, as a required module, I suggest that nothing is printed to the console, all errors should emit up to the calling function; let the application decide if it should care.

**Comment by calmh** [comment]
Sounds reasonable

**Comment by jnovack** [comment]
Is this no longer reasonable?

Will you consider a PR that uses an emit?

**Comment by calmh** [comment]
Sure

**Comment by bangert** [comment]
This one is also closed by 74851c0
